rubygem-katello/rubygem-katello.spec

```diff
@@ -13,7 +13,7 @@ Name:    %{?scl_prefix}rubygem-%{gem_name}
 Summary: Katello
 
 Version: 2.4.0
-Release: 1.nightly%{dist}
+Release: 1.rc1%{dist}
```

Comment:
I asked this in the previous comment, should this and the installer get bumped to 2.rc1 ?